README.md

```diff
@@ -136,3 +136,10 @@ the feature/topic you are going to implement.
 # License
 
 - [GNU General Public License v3.0](LICENSE)
+
+## Pytest marks
+
+Custom pytest marks used in tests:
+* `nightly` - Tests must be runs in Nightly testruns.
+* `not_sbercloud` - Tests must be skips on Sbercloud platform.
```

Comment:
Could you please change the name of the marks without mentioning a specific providers (change to not_vm or something like this).
Thanks.